front50-web/src/main/java/com/netflix/spinnaker/front50/controllers/PipelineController.java

```diff
@@ -267,7 +266,11 @@ private void validatePipeline(final Pipeline pipeline, Boolean staleCheck) {
         pipeline.getApplication(), pipeline.getName().trim(), pipeline.getId());
 
     final GenericValidationErrors errors = new GenericValidationErrors(pipeline);
```

Comment:
I think part of the advantage of `errors` was that you could provide multiple validation errors at once. I'm not sure that's super important.